sbt-compatibility/src/main/scala/sbtcompatibility/SbtCompatibilityInternalKeys.scala

```diff
@@ -12,8 +12,8 @@ trait SbtCompatibilityInternalKeys {
   final val compatibilityUnresolvedWarningConfiguration = taskKey[UnresolvedWarningConfiguration]("")
   final val compatibilityScalaModuleInfo                = taskKey[Option[ScalaModuleInfo]]("")
 
-  final val compatibilityIgnoreSbtDefaultReconciliations = taskKey[Boolean]("")
-  final val compatibilityUseCsrConfigReconciliations     = taskKey[Boolean]("")
+  final val compatibilityIgnoreSbtDefaultReconciliations = settingKey[Boolean]("")
+  final val compatibilityUseCsrConfigReconciliations     = settingKey[Boolean]("")
```

Comment:
Hmm, this means that their value can’t anymore depend on the result of a task. I hope it’s not a problem, but we will need to test a bit more, probably.